config/settings/base.py

```diff
@@ -204,12 +204,13 @@
 INSTALLED_APPS += [
     "django_celery_beat",
 ]
+
+CELERYD_CONCURRENCY = env("CELERYD_CONCURRENCY", default=500)
```

Comment:
For this one, shouldn't it match the number of CPU cores available (default) on the host? Wondering how `500` would affect the number of worker processes spawn per host 🤔

Reply:
But it was a good catch 😉